src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -35,7 +37,12 @@ internal static partial class CFamilyHelper
         public const string CPP_LANGUAGE_KEY = "cpp";
         public const string C_LANGUAGE_KEY = "c";
 
-        internal static readonly string PchFilePath = Path.Combine(Path.GetTempPath(), "SonarLintForVisualStudio.PCH.preamble");
+        internal static string WorkingDirectory { get; } = Path.GetTempPath();
+        internal static readonly string PchFilePath = Path.Combine(WorkingDirectory, "SonarLintForVisualStudio.PCH.preamble");
+        internal static readonly string RequestConfigFilePath = Path.Combine(WorkingDirectory, "sonar-cfamily.request.config");
```

Comment:
@abbas-sabra-sonarsource @duncanp-sonar do you have a suggestion for a better name than `sonar-cfamily.request.config`?

Reply:
I think both files should start with the same prefix - it's simpler to ask a user to "send us the two files starting with X" than to have two completely unrelated file names. Also, the name should make it clear that the file relates to SLVS.

Maybe `SonarLintForVisualStudio.cfamily.request.config`?

Reply:
I think you're looking at the PCH file (`SonarLintForVisualStudio.PCH.preamble`) - the reproducer is named `sonar-cfamily.reproducer`. So what do you think? 😕

Reply:
the name isn't that bad. I think as long as the name starts with the same prefix it is fine(`sonar-cfamily`).
Maybe other suggestion `sonar-cfamily.request.reproducer` as we are trying to reproduce the request. No strong opinion.